CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+2.0.0 October 10, 2016
```

Comment:
I think we also need to include here a point about the breaking changes to `pybutton.Response`